cmd/test/main.go

```diff
@@ -261,6 +274,7 @@ func (flags *TestCommandFlags) AddFlags(cmd *cobra.Command) {
 	// kubeconform flag
 	cmd.Flags().StringArrayVarP(&flags.SchemaLocations, "schema-location", "", []string{}, "Override schemas location search path (can be specified multiple times)")
 	cmd.Flags().BoolVarP(&flags.IgnoreMissingSchemas, "ignore-missing-schemas", "", false, "Ignore missing schemas when executing schema validation step")
+	cmd.Flags().StringVarP(&flags.SaveRendered, "save-rendered", "", "", "Save The rendered file. needs to be a path to a directory")
```

Comment:
does this have to be a path to a directory? or could it also be a path to a yaml file?

Reply:
also, should this be global path only? or also local path?